istioctl/cmd/analyze.go

```diff
@@ -288,6 +306,9 @@ istioctl analyze -L
 		"Overrides the mesh config values to use for analysis.")
 	analysisCmd.PersistentFlags().BoolVar(&allNamespaces, "all-namespaces", false,
 		"Analyze all namespaces")
+	analysisCmd.PersistentFlags().StringArrayVar(&suppress, "suppress", []string{},
+		"Suppress reporting a message code on a specific resource. Values are supplied in the form "+
```

Comment:
Use backticks to create a raw string and you don't need to escape the backslash.

Reply:
If I use backticks though, I'm forced to make the entire string a single line as newlines are preserved, right?

Reply:
You can use + to concat `` strings together just like "" strings, should be fine.

Reply:
🤦‍♂ I'm dense. Fixed.